core/data/stories.md

```diff
@@ -266,8 +258,6 @@
   - form{"name": null}
   - slot{"self_assess_done": true}
   - slot{"symptoms": "mild"}
-  - utter_ask_symptoms_worsened
-* deny
```

Comment:
now that this branching is out of the way, is there any dupplicated stories we should remove?

Reply:
I didn't see it but yes, there is one I can remove. Others show different cases of following QA